frontend/public/components/masthead-toolbar.jsx

```diff
@@ -260,19 +295,21 @@ class MastheadToolbar_ extends React.Component {
             {flags[FLAGS.OPENSHIFT] && <ToolbarItem>
               <Dropdown
                 isPlain
+                key="applauncher"
```

Comment:
Why `key`?

Reply:
Removed it here, only needed it in `<DropdownItem ../>`'s